Makefile.am

```diff
@@ -1,3 +1,7 @@
+ACLOCAL_AMFLAGS = -I m4
+#AM_CPPFLAGS = -I$(top_builddir)/gnulib -I$(top_srcdir)/gnulib
+DEFAULT_INCLUDES = -I$(top_srcdir)/gnulib
```

Comment:
I don't like this change. I would like to keep the code representing "WE ARE USING GNULIB" explicitly.
How about using following code?
```
GNULIB_CPPFLAGS = I$(top_srcdir)/gnulib
```

So ... *B*

Reply:
Using DEFAULT_INCLUDES reduces the lines of the build script, however, it breaks the rhythm (or pattern) of lines.

Reply:
I used `DEFAULT_INCLUDES` because I thought `-I$(top_srcdir)/gnulib` had to be defined earlier than other include paths.
It is because  `-I$(top_srcdir)/gnulib` may override system include files. But other include paths do not override system include files.

I will fix them as you suggested.

Reply:
I've fixed.